network-store-server/src/main/resources/db/changelog/changesets/changelog_2022-09-28T21:30:00Z.xml

```diff
@@ -0,0 +1,227 @@
+            <column name="phasetapchangerterminalrefconnectableid3" type="VARCHAR(255)"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangerterminalrefside3" type="TEXT"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangerregulationmode3" type="TEXT"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="phasetapchangerregulationvalue3" type="FLOAT8"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangerlowtapposition3" type="INTEGER"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangertapposition3" type="INTEGER"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangerregulating3" type="BOOLEAN"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangertargetdeadband3" type="FLOAT8"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangerterminalrefconnectableid3" type="VARCHAR(255)"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangerterminalrefside3" type="TEXT"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangerloadtapchangingcapabilities3" type="BOOLEAN"/>
+        </addColumn>
+        <addColumn tableName="threewindingstransformer">
+            <column name="ratiotapchangertargetv3" type="FLOAT8"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="cboutier" id="1664174319707-3">
+        <createTable tableName="tapchangerstep">
+            <column name="equipmentid" type="VARCHAR(255)"/>
+            <column name="equipmenttype" type="VARCHAR(255)"/>
+            <column name="networkuuid" type="UUID"/>
+            <column name="variantnum" type="INTEGER"/>
+            <column name="index" type="INTEGER"/>
+            <column name="side" type="INTEGER"/>
+            <column name="tapchangertype" type="VARCHAR(50)"/>
+            <column name="rho" type="FLOAT8"/>
+            <column name="r" type="FLOAT8"/>
+            <column name="x" type="FLOAT8"/>
+            <column name="g" type="FLOAT8"/>
+            <column name="b" type="FLOAT8"/>
+            <column name="alpha" type="FLOAT8"/>
+        </createTable>
+    </changeSet>
+    <changeSet author="cboutier" id="1664174319707-4">
+        <addPrimaryKey columnNames="equipmentid, networkuuid, variantnum, index, side, tapchangertype" constraintName="tapchangerstep_pkey" tableName="tapchangerstep"/>
+    </changeSet>
+</databaseChangeLog>
```

Comment:
Don't forget to add the INDEX for the tapchangerstep table.

Reply:
fixed